app/test_split_integer.py

```diff
@@ -2,20 +2,28 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    result = split_integer(8, 2)
+    assert sum(result) == 8, (f"Expected sum to be 8, but got {sum(result)} "
+                              f"for result {result}")
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    result = split_integer(6, 2)
+    assert result == [3, 3], f"Expected [3, 3] but got {result}"
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    result = split_integer(1, 1)
+    assert result == [1], f"Expected result to be 1, but got {result}"
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    result = split_integer(5, 3)
+    assert result == sorted(result), f"Expected result to be sorted but got {result}"
 
 
 def test_should_add_zeros_when_value_is_less_than_number_of_parts() -> None:
-    pass
+    result = split_integer(1, 4)
+    assert result == [0, 0, 0, 1], f"Expected result to be equal to 1, but got {result}"
```

Comment:
The assertion message in `test_should_add_zeros_when_value_is_less_than_number_of_parts` is misleading. The message states 'Expected result to be equal to 1', but the expected result is `[0, 0, 0, 1]`. Consider updating the message to accurately reflect the expected result.